swap_meet/clothing.py

```diff
@@ -1,2 +1,15 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition = 0):
+        '''
+        Constructor of an object of parent class. Its parameter condition has a default value of 0
+        '''
+        super().__init__(category = "Clothing", condition = condition)
```

Comment:
👍  Good work in ensuring that all `Clothing` instances have a `category` of `"Clothing"` by passing the value into the parent constructor.